packages/dev/test/StakeManager.test.ts

```diff
@@ -476,4 +476,23 @@ contract('StakeManager', function ([burnAddress, relayManager, anyRelayHub, owne
       testOwnerCanStake()
     })
   })
+
+  context('#setBurnAddress', function () {
+    before(async function () {
+      stakeManager = await StakeManager.new(defaultEnvironment.maxUnstakeDelay, constants.BURN_ADDRESS)
+    })
+
+    it('should allow owner to change burn address', async function () {
```

Comment:
nitpic: there is a code to make sure burnAddress is not zero, but no test for it (not critical..)